linera-storage/src/rocksdb.rs

```diff
@@ -3,28 +3,28 @@
 
 use crate::{chain_guards::ChainGuards, DbStore, DbStoreClient};
 use linera_execution::WasmRuntime;
-use linera_views::rocksdb::RocksdbClient;
+use linera_views::rocksdb::RocksDbClient;
```

Comment:
Can we deal with such a large-scale renaming in another PR? Also, technically, if we go for `RocksDb`, then the module should be called `rocks_db`.

Reply:
Right… sorry, I didn't realize how big that change would be.

Reply:
It was asked by Andreas, let me revert then.